scripts/convert_gen2_to_gen3_hsc.sh

```diff
@@ -8,7 +8,30 @@
 
 # Run the gen2 to gen3 conversion code
 # This conversion is done in-place so the same path is used for the gen2 and gen3 repos.
```

Comment:
I think you need to configure the script to check exit status of each command automatically, otherwise you'll get cascading errors as every subsequent command fails.

Reply:
I'm not much of a shell-scripter, so I don't know how to do this.  Any tips?

Reply:
Add this to the top of the script:
```bash
set -i
```

Reply:
Noting that I updated the previous comment since I had a typo in it initially.